src/components/webview/MainWebviewProvider.ts

```diff
@@ -48,7 +48,8 @@ import { CaseManager } from '../../cases/caseManager';
 import { SourceControlService } from '../sourceControl';
 
 export const ROOT_ELEMENT_HASH: ElementHash = '' as ElementHash;
-const ROOT_FOLDER_KEY = '/';
+const ROOT_FOLDER_KEY =
```

Comment:
this should be evaluated before doing an operation, not when the extension is being initialized